tests/linters_output/gitleaks.json

```diff
@@ -0,0 +1,34 @@
+[
+        {
+           "line": "    aws_access_key_id='AKIAIO5FODNN7EXAMPLE',",
+           "lineNumber": 5,
+           "offender": "AKIAIO5FODNN7EXAMPLE",
+           "commit": "cdb55093ab0c59fd16d2bef32453d7d7c1c0a577",
+           "repo": ".",
+           "repoURL": "",
+           "leakURL": "",
+           "rule": "AWS Access Key",
+           "commitMessage": "adding entropy test finding\n",
+           "author": "bob s",
+           "email": "bob@example.com",
+           "file": "relative/path/to/output",
+           "date": "2021-03-01T11:23:31-05:00",
+           "tags": "key, AWS"
+        },
+        {
+         "line": "            \"line\": \"-----BEGIN PRIVATE KEY-----\",",
```

Comment:
I still think it's confusing for a future maintainer of this project to use a false positive test case here.

I'm assuming you got this from running gitleaks on the current file (gitleaks.json), which is a FP, because there's no actual private key material here.

If someone had checked in a _real_ key, the line would start with just `-----BEGIN PRIVATE KEY-----`. (In other words, this entire line should just be
```
         "line": "-----BEGIN PRIVATE KEY-----",
```
).

Reply:
So you want me to change this line to `-----BEGIN PRIVATE KEY-----` or add extra file with private key and then write test case around that?

Reply:
Just change the line. I don't think you need to add extra tests or make any changes to the existing test.